app/views/admin/finance/statements/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<% page_data(title: @statement.page_title) %>
+
+<%=
+  govuk_summary_list do |sl|
+    sl.with_row do |row|
+      row.with_key(text: "Lead provider")
+      row.with_value(text: govuk_link_to(@statement.lead_provider_name, '#', text_colour: true))
+    end
+
+    sl.with_row do |row|
+      row.with_key(text: "Status")
+      row.with_value(text: govuk_tag(**@statement.status_tag_kwargs))
+    end
+
+    sl.with_row do |row|
```

Comment:
could we add the contract year here as well?

Reply:
Hmm yeah - just with a placeholder value you mean?

Reply:
sorry meant `registration_period_year`